xblock/test/test_parsing.py

```diff
@@ -19,7 +19,7 @@
 
 def get_namespace_attrs():
     """ Returns string suitable to be used as an xmlns parameters in XBlock XML representation """
-    return " ".join('xmlns:{}="{}"'.format(k, v) for k, v in XML_NAMESPACES.items())
+    return " ".join('xmlns:{}="{}"'.format(k, v) for k, v in sorted(XML_NAMESPACES.items()))
```

Comment:
@cpennington @nedbat See the current test failure to see the test flakiness here. This was passing prior to my current commit (which is unrelated to this test).

I am en route to SFO right now, so I don't have a great way to test locally (I pushed and relied on Travis), but I believe I addressed all issues? 

I'm not 100% certain about optional requires -- the way pip is handling it on my system is completely broken, and if one of you were to confirm it works, I'd be grateful.